src/main/java/com/uipath/uipathpackage/models/JobOptions.java

```diff
@@ -6,7 +6,7 @@
 public class JobOptions extends AuthenticatedOptions {
     private String processName;
     private String parametersFilePath;
-    private final String telemetryOrigin = "Jenkins";
+    private static final String telemetryOrigin = "Jenkins";
```

Comment:
I can see this line, replicated in every other file since this is static and final signaling it's a constant.
it can be extracted out and put in Util class, and can be referenced from there.

Reply:
This has been taken care in this [PR ](https://github.com/jenkinsci/uipath-automation-package-plugin/pull/139) which is raised against this branch.